spec/forms/webauthn_verification_form_spec.rb

```diff
@@ -62,28 +67,23 @@
     end
 
     context 'when the input is invalid' do
-      context 'when user is missing' do
```

Comment:
I'm going to double-check this, but I can't see any reason why we'd ever be initializing the form with a `nil` user.